### DISCUSSION TITLE
Users last login

### DISCUSSION THREAD

**Discussion by innovate-support** [Security-Onion-Solutions/securityonion#5222] [Ideas] [answered]
There's a Users section on the web interface but it really isn't useful at all in it's current state.  It would be great if the First Name and Last Name could be configured, and also if it showed the last login date/time.  We have a number of Analysts and I'd like a little more info displayed.

**Comment by dougburks** [comment, marked as answer]
> There's a Users section on the web interface but it really isn't useful at all in it's current state.

Please consider your tone when posting statements like this.  It comes across in such a way that makes us less likely to want to help you. Please review the Etiquette section of https://github.com/Security-Onion-Solutions/securityonion/discussions/1720.

> It would be great if the First Name and Last Name could be configured, and also if it showed the last login date/time. We have a number of Analysts and I'd like a little more info displayed.

Yep, this has been the plan all along, we just haven't had time to get to it yet.  We're working as hard as we can given the limited size of our team.  If you would like to help expedite this process, we will gladly accept pull requests or referrals for an [experienced Golang developer](https://blog.securityonion.net/2021/08/security-onion-solutions-is-looking-for.html). 

Thanks!
